Description: Fix 'make check' for root
Origin: upstream, commit:c67913c974b9a1d95f6e00a616074a9eea175920
Last-Update: 2018-07-18
Index: libvirt-dbus/tests/libvirttest.py
===================================================================
--- libvirt-dbus.orig/tests/libvirttest.py
+++ libvirt-dbus/tests/libvirttest.py
@@ -33,7 +33,7 @@ class BaseTestClass():
         """Start libvirt-dbus for each test function
         """
         os.environ['LIBVIRT_DEBUG'] = '3'
-        self.libvirt_dbus = subprocess.Popen([exe])
+        self.libvirt_dbus = subprocess.Popen([exe, '--session'])
         self.bus = dbus.SessionBus()
 
         for i in range(10):
